core/src/main/java/com/klaytn/caver/transaction/type/EthereumDynamicFee.java

```diff
@@ -0,0 +1,794 @@
+     * @param value                The amount of KLAY in peb to be transferred.
+     */
+    public EthereumDynamicFee(Klay klaytnCall, String from, String nonce, String gas, String maxPriorityFeePerGas, String maxFeePerGas, String chainId, List<SignatureData> signatures, String to, String input, String value, AccessList accessList) {
+        super(
+                klaytnCall,
+                TransactionType.TxTypeEthereumDynamicFee.toString(),
+                from,
+                nonce,
+                gas,
+                chainId,
+                signatures
+        );
+        setTo(to);
+        setValue(value);
+        setInput(input);
+        setAccessList(accessList);
+        setMaxPriorityFeePerGas(maxPriorityFeePerGas);
+        setMaxFeePerGas(maxFeePerGas);
+    }
+
+    /**
+     * Getter function for maxPriorityFeePerGas
+     * @return String
+     */
+    public String getMaxPriorityFeePerGas() {
+        return maxPriorityFeePerGas;
+    }
+
+    /**
+     * Setter function for maxPriorityFeePerGas.
+     * @param maxPriorityFeePerGas Max priority fee per gas.
+     */
+    public void setMaxPriorityFeePerGas(String maxPriorityFeePerGas) {
+        if(maxPriorityFeePerGas == null || maxPriorityFeePerGas.isEmpty() || maxPriorityFeePerGas.equals("0x")) {
+            maxPriorityFeePerGas = "0x";
+        }
+
+        if(!maxPriorityFeePerGas.equals("0x") && !Utils.isNumber(maxPriorityFeePerGas)) {
+            throw new IllegalArgumentException("Invalid maxPriorityFeePerGas. : " + maxPriorityFeePerGas);
+        }
+
+        this.maxPriorityFeePerGas = maxPriorityFeePerGas;
+    }
+
+    /**
+     * Setter function for maxPriorityFeePerGas.
+     * @param maxPriorityFeePerGas Max priority fee per gas.
+     */
+    public void setMaxPriorityFeePerGas(BigInteger maxPriorityFeePerGas) {
+        setMaxPriorityFeePerGas(Numeric.toHexStringWithPrefix(maxPriorityFeePerGas));
+    }
+
+    /**
+     * Getter function for maxFeePerGas.
+     * @return String
+     */
+    public String getMaxFeePerGas() {
+        return maxFeePerGas;
+    }
+
+    /**
+     * Setter function for maxFeePerGas.
+     * @param maxFeePerGas Max fee per gas.
+     */
+    public void setMaxFeePerGas(String maxFeePerGas) {
+        if(maxFeePerGas == null || maxFeePerGas.isEmpty() || maxFeePerGas.equals("0x")) {
+            maxFeePerGas = "0x";
+        }
+
+        if(!maxFeePerGas.equals("0x") && !Utils.isNumber(maxFeePerGas)) {
+            throw new IllegalArgumentException("Invalid maxFeePerGas. : " + maxFeePerGas);
+        }
+
+        this.maxFeePerGas = maxFeePerGas;
+    }
+
+    /**
+     * Setter function for gas price.
+     * @param maxFeePerGas Max fee per gas.
+     */
+    public void setMaxFeePerGas(BigInteger maxFeePerGas) {
+        setMaxFeePerGas(Numeric.toHexStringWithPrefix(maxFeePerGas));
+    }
+
+
+    @Override
+    public String getRLPEncoding() {
```

Comment:
Please add some comments for `getRLPEncoding` and `getCommonRLPEncodingForSignature` functions.